### PR TITLE
Add tables for the compressed Nsigma values

### PR DESCRIPTION
**Pull request by njacazio**
- Add dedicated tasks

### CODE REVIEW

File: Analysis/DataModel/include/AnalysisDataModel/PID/PIDResponse.h [hunk 1]
```diff
@@ -74,12 +74,41 @@ DECLARE_SOA_COLUMN(TOFNSigmaHe, tofNSigmaHe, float);
 DECLARE_SOA_COLUMN(TOFNSigmaAl, tofNSigmaAl, float);
 } // namespace pidtof
 
+namespace pidtof_tiny
+{
+constexpr int nbins = 256;
```
Comment (jgrosseo) [inline Analysis/DataModel/include/AnalysisDataModel/PID/PIDResponse.h]: is there a constexpr sizeof? then we could derive it from int8_t, or make even int8_t a define here

Comment (njacazio) [reply Analysis/DataModel/include/AnalysisDataModel/PID/PIDResponse.h]: uhm that's a good point !

File: Analysis/DataModel/include/AnalysisDataModel/PID/PIDResponse.h [hunk 2]
```diff
@@ -74,12 +74,61 @@ DECLARE_SOA_COLUMN(TOFNSigmaHe, tofNSigmaHe, float);
...
+DECLARE_SOA_COLUMN(TOFNSigmaStoreKa, tofNSigmaStoreKa, binned_nsigma);
+DECLARE_SOA_COLUMN(TOFNSigmaStorePr, tofNSigmaStorePr, binned_nsigma);
+DECLARE_SOA_COLUMN(TOFNSigmaStoreDe, tofNSigmaStoreDe, binned_nsigma);
+DECLARE_SOA_COLUMN(TOFNSigmaStoreTr, tofNSigmaStoreTr, binned_nsigma);
+DECLARE_SOA_COLUMN(TOFNSigmaStoreHe, tofNSigmaStoreHe, binned_nsigma);
+DECLARE_SOA_COLUMN(TOFNSigmaStoreAl, tofNSigmaStoreAl, binned_nsigma);
+// NSigma with reduced size in [bin_min, bin_max] bin size bin_width
+DECLARE_SOA_DYNAMIC_COLUMN(TOFNSigmaEl, tofNSigmaEl, [](float nsigma_binned) -> float {
+  return bin_min + bin_width * nsigma_binned;
+});
```
Comment (jgrosseo) [inline Analysis/DataModel/include/AnalysisDataModel/PID/PIDResponse.h]: Let's add a macro that does this
UNWRAP_COLUMN(TOFNSigmaEl, tofNSigmaEl, bin_min, bin_width)

File: Analysis/Tasks/pidTOF_tiny.cxx [hunk 3]
```diff
@@ -0,0 +1,113 @@
...
+      if (resp_PID.GetExpectedSigma(resp, trk.collision(), trk) <= o2::aod::pidtof_tiny::bin_min) {
+        tofpid(resp_PID.GetExpectedSigma(resp, trk.collision(), trk),
+               0);
+      } else if (resp_PID.GetExpectedSigma(resp, trk.collision(), trk) >= o2::aod::pidtof_tiny::bin_max) {
+        tofpid(resp_PID.GetExpectedSigma(resp, trk.collision(), trk),
+               o2::aod::pidtof_tiny::nbins);
+      } else {
+        tofpid(resp_PID.GetExpectedSigma(resp, trk.collision(), trk),
+               (resp_PID.GetSeparation(resp, trk.collision(), trk) - o2::aod::pidtof_tiny::bin_min) / (o2::aod::pidtof_tiny::bin_width)*o2::aod::pidtof_tiny::nbins);
+      }
```
Comment (jgrosseo) [inline Analysis/Tasks/pidTOF_tiny.cxx:88]: In this way the resp_PID.GetExpectedSigma is computed up to three times. Better put it in a variable before. Does this conflict with an earlier comment? ;-)

File: Analysis/DataModel/include/AnalysisDataModel/PID/PIDResponse.h (hunk 2 above)
Comment (njacazio) [reply Analysis/DataModel/include/AnalysisDataModel/PID/PIDResponse.h]: This is done!

File: Analysis/Tasks/pidTOF_tiny.cxx (hunk 3 above)
Comment (njacazio) [reply Analysis/Tasks/pidTOF_tiny.cxx:88]: I don't know what I was thinking, I probably got confused and thought it was coming from a table ;) However, it is fixed now

**Review by jgrosseo — APPROVED** [review]

**Pull request merged**